browser-test/src/support/applicant_questions.ts

```diff
@@ -519,9 +519,20 @@ export class ApplicantQuestions {
     ).toEqual(0)
   }
 
-  async expectVerifyAddressPage(validAddress: boolean) {
-    const header = validAddress ? 'Verify address' : 'No valid address found'
-    expect(await this.page.innerText('h2')).toContain(header)
+  async expectVerifyAddressPage(hasAddressSuggestions: boolean) {
+    expect(await this.page.innerText('h2')).toContain('Confirm your address')
+    if (hasAddressSuggestions) {
+      // Note: If there's only one suggestion, the heading will be "Suggested address"
+      // not "Suggested addresses". But, our browser setup always returns multiple
+      // suggestions so we can safely assert the heading is always "Suggested addresses".
+      await expect(
+        this.page.getByRole('heading', {name: 'Suggested addresses'}),
+      ).toBeVisible()
```

Comment:
optional: You should be able to pass a visible boolean to toBeVisible so you could do await expect(blah).toBeVisible(hasAddressSuggestions) https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-visible so you don't need this if/else

Reply:
Ooh nice, updated.